sim/core/consumes.go

```diff
@@ -71,104 +82,198 @@ func ConsumesStats(c proto.Consumes) stats.Stats {
 	return s
 }
 
-func TryActivateDrums(sim *Simulation, character *Character) {
-	if character.IsOnCD(MagicIDDrums, sim.CurrentTime) {
-		return
-	}
+// Adds drums as a major cooldown to the character, if it's being used.
+func registerDrumsCD(agent Agent, consumes proto.Consumes) {
```

Comment:
Because this is referencing "party buffs" I assume this would only be used in an individual sim and not a "raid sim"?